### PR TITLE
Migrate to release-it v11

### PR DESCRIPTION
**Pull request by webpro**
**Summary**

As I was working on release-it v11, I tried a few projects in the wild and yours was one of them. So why not open a PR if I have the changes already :)

**Test plan**

You can check out my PR and run `npm run release` (without actually releasing) to see whether it does the job for you.

### CODE REVIEW

**Comment by satya164** [comment]
Thank you for the PR :D

I'll give it a try soon.

**Comment by thymikee** [comment]
@webpro seems like the latest is 12.2 now, would you like to give it a try? :)

**Comment by webpro** [comment]
There are [no breaking changes here](https://github.com/release-it/release-it/blob/master/CHANGELOG.md#v12), so can be updated whenever it fits.

File: package.json [hunk 1]
```diff
@@ -83,6 +82,7 @@
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
     "@babel/register": "^7.0.0",
     "@emotion/is-prop-valid": "^0.7.3",
+    "@release-it/conventional-changelog": "^1.0.0",
```
Comment (thymikee) [inline package.json]: this should be moved to devDependencies

Comment (webpro) [reply package.json]: Oops! Also updated the versions.

**Comment by thymikee** [comment]
Looks like yarn is still complaining on outdated lockfile

**Comment by thymikee** [comment]
Thanks! Btw, is there a reliable way to use `release-it` with monorepo? We want to split linaria into separate packages and that would be helpful

**Comment by webpro** [comment]
There's no built-in way to e.g. `npm publish` each package separately. Depending on your needs, this might be a fairly simply script running release-it from each package dir. I've discussed this topic with @danderson00 in https://github.com/release-it/release-it/issues/516 and he started https://github.com/danderson00/release-em. Happy to discuss this topic and/or help out, especially interested in generic use cases.

**Comment by danderson00** [comment]
Hey @thymikee, I suspect our requirements are similar, mostly focused around taking the pain out of managing versions and publishing multiple npm packages. Have a look at release-em, it's new and a bit immature, but I'd be more than happy to work with you to get it to do what you need reliably.

**Pull request merged**